windows/src/test/unit-tests/kmcomp/test_5963_start_of_sentence.kmn

```diff
@@ -21,7 +21,7 @@ store(digit) '0'..'9'
     if(&newLayer = "") if(&layer = 'numeric') any(digit) > context
 
     c no other changes, so detect sentence or layer change
-    if(&newLayer = 0") > use(nextLayer)
+    if(&newLayer = "") > use(nextLayer)
```

Comment:
Does `&newLayer` depend on `store(&VERSION) '15.0'`?

https://help.keyman.com/developer/language/reference/newlayer#toc-version-history

Reply:
This is an interesting one. We could make it require version 15.0, as the newContext/postKeystroke functionality won't work on earlier versions, but you could legitimately run the keyboard with reduced functionality on earlier versions. What's more, version 15.0 will pick up the new functionality correctly even when the keyboard is tagged with an earlier version number -- it's not a breaking change.

(The min version flag is really around breaking changes -- data that may cause an earlier version of Keyman to fail to load the keyboard or cause other errors after load. This was not 100% consistent through v6.0-v9.0, but it has been pretty consistent in recent versions.)

Given this, the compiler says, hey, that's fine. This keyboard will still work with earlier versions of Keyman and won't cause errors. So it assigns v9.0 to the keyboard. I think that's appropriate behaviour.

But it's a little misleading for keyboard authors!